listenbrainz_spark/stats/__init__.py

```diff
@@ -44,3 +44,7 @@ def replace_days(day):
 def adjust_months(date, months):
```

Comment:
Both of these functions bother me a little in that they are not very flexible. What if we didn't want a date range that led up to now? What if we wanted listens between arbitrary dates?

Reply:
IMO, listens between arbitrary date range can be fetched by changing the begin_date and end_date, these functions just serve the purpose of going back and ahead in time.